portal/src/main/webapp/js/src/patient-view/util/ClinicalAttributesUtil.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+/**
+ * Functions for dealing with clinical attributes.
+ */
+window.ClinicalAttributesUtil = (function() {
+  /**
+   * Clean clinical attributes. Useful for rounding numbers, or other types of
+   * data cleanup steps. Probably differs per institution.
+   * @param {object} clinicalData - key/value pairs of clinical data
+   */
+  function clean(clinicalData) {
+    // Shallow Copy clinicalData
+    var cleanClinicalData = $.extend({}, clinicalData);
+    var NULL_VALUES = [
```

Comment:
Btw @pieterlukasse @fedde-s I agree that these `NULL_VALUES` should prolly be at the java layer (or preferably T step). When I coded this there were still some of these values in the db. Not sure about now, it might be possible to get rid of it.